.github/workflows/staging.yml

```diff
@@ -0,0 +1,89 @@
+name: Build & Staging
+on:
+  push:
+#    branches:
```

Comment:
why commented?

Reply:
This was commented to test out that deployment to staging works before we push to the `main` branch. I'll uncomment it now that it is finally working :D